lib/ocpp/v201/average_meter_values.cpp

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+
+#include <everest/logging.hpp>
+#include <everest/timer.hpp>
+#include <ocpp/v201/average_meter_values.hpp>
+
+namespace ocpp {
+namespace v201 {
+AverageMeterValues::AverageMeterValues() {
+    EVLOG_debug << "constructor";
+    // reset the values
+}
+void AverageMeterValues::clear_values() {
+    EVLOG_debug << " Clearing the aligned values";
+    // reset the values
+    this->aligned_meter_values.clear();
+    this->averaged_meter_values.sampledValue.clear();
+}
+
+void AverageMeterValues::set_values(const MeterValue& meter_value) {
+    EVLOG_debug << " Setting the aligned values";
```

Comment:
Maybe remove these log lines? Even for debugging this is not really useful IHMO. You'd just get spammed with this line, maybe even a couple of times a second with a fast meter.

At the very least remove the log from the constructor. It is assumed a constructor just works so does not really need logging like this.